lms/views/lti/deep_linking.py

```diff
@@ -133,6 +134,11 @@ def file_picker_to_form_fields_v13(self):
             ],
         }
 
+        if title := assignment_configuration.get("title"):
```

Comment:
Not much going on here.

We don't want to include a title key when there's no value so there's one conditional here and in the tests.